src/common/utils/formatter/formatter.ts

```diff
@@ -108,7 +108,11 @@ export function formatValue(value: any, timezone?: Timezone, displayYear?: Displ
 }
 
 export function formatDateWithTimezone(value: Date, timezone: Timezone) {
-  return moment.tz(value, timezone.toString()).format("YYYY-MM-DD HH:mm:ss (z)");
+  return moment.tz(value, timezone.toString()).format("YYYY-MM-DD HH:mm:ss");
```

Comment:
You just removed the timezone from `formatDateWithTimezone` :) Would make sense to rename the two functions.